xml/admin_monitoring_alerting.xml

```diff
@@ -99,87 +99,20 @@
   deploying services.
  </para>
  <sect1 xml:id="monitoring-stack-deploy">
-  <title>Deploying Monitoring With &cephadm;</title>
+  <title>Deploying the Monitoring Stack</title>
 
-  <para>
-   To deploy the monitoring stack you need to either use a YAML configuration
-   file or the CLI. Whichever method you will use, you need to ensure the
-   &prometheusmgr; is enabled.
-  </para>
-
-  <para>
-   The &prometheusmgr; needs to be enabled using the following command:
-  </para>
-
-<screen>&prompt.cephuser;ceph mgr module enable prometheus</screen>
-
-  <note>
-   <para>
-    Ensure this command is run before &prometheus; is deployed. If the command
-    was not run before the deployment, you must redeploy &prometheus; to update
-    Prometheus' configuration:
-   </para>
-<screen>&prompt.cephuser;ceph orch redeploy prometheus</screen>
-  </note>
```

Comment:
This note is not part of the deployment guide. It might make sense to add it there. If, for some reason, the prometheus mgr module is not or wasn't enabled, the `redeploy` command needs to be used to fix the prometheus configuration. Without this hint, that might become tedious.